### PR TITLE
Schedule `ip-reconciler` CronJob on infra nodes

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 1]
```diff
@@ -6,14 +6,28 @@ The parent key for all of the following parameters is `openshift4_networking`.
...
+[horizontal]
+type:: dictionary
+default::
++
+[source,yaml]
+----
+node-role.kubernetes.io/infra: ''
+----
+
+Node selector for the non-DaemonSet OpenShift Multus workloads.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc:25]:
```suggestion
Node selector for the non-DaemonSet OpenShift networking workloads.
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 2]
```diff
@@ -6,14 +6,28 @@ The parent key for all of the following parameters is `openshift4_networking`.
 
 [horizontal]
 type:: string
-default:: `syn-openshift4-networking`
+default:: `openshift-multus`
 
-The namespace in which to deploy this component.
+The namespace in which OpenShift Multus is deployed.
```
[the remark this answers is not in the document]
Comment (bastjan) [reply docs/modules/ROOT/pages/references/parameters.adoc]: I added a slightly more generic version. How do you like it?

File: class/defaults.yml [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+    defaultNodeSelector:
+      node-role.kubernetes.io/infra: ''
+
+    patches:
+      ip-reconciler:
+        target:
+          kind: CronJob
+          apiVersion: batch/v1
+          name: ip-reconciler
+          namespace: openshift-multus
```
Comment (simu) [inline class/defaults.yml:14]: I wonder if we can remove the repetition here with something like:

```suggestion
      openshift-multus:
        ip-reconciler:
          target:
            kind: CronJob
            apiVersion: batch/v1
```

Not sure how we'd handle cluster-scoped patches then, but I don't think that's something we need to worry about at this point.

**Review by simu — APPROVED** [review]
LGTM now, I'd probably opt to move namespace and object name into the parameter hierarchy instead of using a "dead" key, but both variants have their advantages.

File: class/defaults.yml (hunk 3 above)
Comment (simu) [reply class/defaults.yml:14]: I have no hard preference here, we can just leave `name` and `namespace` in `target`

**Pull request merged**